pkgs/tools/backup/awsbck/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, rustPlatform
+, installShellFiles
+, Security
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "awsbck";
+  version = "0.3.2";
+
+  src = fetchFromGitHub {
+    owner = "beeb";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-4iFPHMCWKOfwqdjCLQqWHSs5SwXi+K2sQu75ecsolSs=";
```

Comment:
```suggestion
    hash = "sha256-4iFPHMCWKOfwqdjCLQqWHSs5SwXi+K2sQu75ecsolSs=";
```

Reply:
Done! Sorry for the rebases and force pushes due to formatting shenanigans, finally got to the state I wanted.